openreview/conference/builder.py

```diff
@@ -1016,8 +1016,9 @@ def get_signatures(self, conference, number):
 
 class CommentStage(object):
 
-    def __init__(self, start_date = None, allow_public_comments = False, anonymous = False, unsubmitted_reviewers = False, reader_selection = False, email_pcs = False, authors=False):
-        self.name = 'Comment'
+    def __init__(self, official_name = None, start_date = None, allow_public_comments = False, anonymous = False, unsubmitted_reviewers = False, reader_selection = False, email_pcs = False, authors=False):
```

Comment:
Can we call the parameter `official_comment_name` instead?

Reply:
it is already a comment stage, why do you need to add the work comment?

Reply:
Official_name sounds like the name of some official involved with the paper. Maybe invitation_name would be better?

Reply:
I changed the value to `official_comment_name`